pkg/config/config.go

```diff
@@ -147,6 +147,22 @@ type Homebrew struct {
 	Goarm                 string               `yaml:"goarm,omitempty"`
 }
 
+// Krew contains the krew section.
+type Krew struct {
```

Comment:
Alternative thought: What if the user provided a template YAML file here in each krews[], and you filled it up by rendering the template? This is what [krew-release-bot](https://github.com/rajatjindal/krew-release-bot) does. That way, you get to preserve the shape of the YAML and only edit the pieces you're interested.

I understand Goreleaser helps a lot here by automatically generating `spec.platforms` but what if user still provided a YAML file that just lacked `spec.platforms` but most everything else like description etc was in place?

I suggest templating this because (1) we expect field ordering consistency between YAMLs of different plugins (2) I am not sure if stuff like caveats/description fields belong to .goreleaser.yml from a logical standpoint.

Reply:
on the point 1, we can order the artifacts by goos/goarch so they are always in the same order... 

on point 2, we already have those fields for other things like homebrew... so I think it should be ok 🤔